### PR TITLE
Added section about plugin usage in Mill

### PR DESCRIPTION
**Pull request by lefou**

### CODE REVIEW

File: _overviews/plugins/index.md [hunk 1]
```diff
@@ -243,6 +243,38 @@ For more details, see [Compiler Plugin
...
+
+Example:
+
+```scala
+// build.sc
+import mill._, mill.scalalib._
+
+object foo extends ScalaModule {
+  // Add the compiler plugin divbyzero in version 1.0
+  def scalacPluginIvyDeps = Agg(ivy"org.divbyzero:::"divbyzero:1.0")
```
Comment (julienrf) [inline _overviews/plugins/index.md:261]:
```suggestion
  def scalacPluginIvyDeps = Agg(ivy"org.divbyzero:::divbyzero:1.0")
```
?

**Review by julienrf — COMMENTED** [review]
Thank you @lefou! I think there is a typo. Otherwise, this looks good to me.

File: _overviews/plugins/index.md (hunk 1 above)
Comment (lefou) [reply _overviews/plugins/index.md:261]: Oh, yes. Thanks!

Comment (lefou) [reply _overviews/plugins/index.md:261]:
```suggestion
  def scalacPluginIvyDeps = Agg(ivy"org.divbyzero:::divbyzero:1.0")
```

**Review by julienrf — APPROVED** [review]

**Pull request merged**